draft-ietf-quic-qpack.md

```diff
@@ -95,17 +95,29 @@ reintroduced by HPACK when the loss includes a HEADERS frame.
 
 ## Avoiding Head-of-Line Blocking in HTTP/QUIC {#overview-hol-avoidance}
 
-In the example above, the second stream contained a reference to data
-which might not yet have been processed by the recipient.  Such references
-are called "vulnerable," because the loss of a different packet can keep
-the reference from being usable.
-
-The encoder can choose on a per-header-block basis whether to favor higher
+Because QUIC does not guarantee order between data on different streams, a
+reference might be received on a request stream to an entry which is not yet in
```

Comment:
request streams aren't the only affected streams

"a header block might reference an entry in the dynamic table that has not yet been received."